src/app/(guest)/login/page.tsx

```diff
@@ -40,19 +49,27 @@ const Page = () => {
       </div>
       <div className="flex flex-1 flex-col justify-center bg-slate-100 px-20">
         <h3 className="mb-6 text-2xl">Sign in</h3>
-        <Input
-          name="email"
-          type="email"
-          label="Email"
-          placeholder="Enter your email"
-        />
-        <Input
-          name="password"
-          type="password"
-          label="Password"
-          placeholder="Enter your password"
-        />
-        <Button onClick={submitForm}>Sign in</Button>
+        <form onSubmit={formik.handleSubmit}>
+          <Input
+            name="email"
+            type="email"
+            label="Email"
+            placeholder="Enter your email"
+            value={formik.values.email}
+            onChange={formik.handleChange}
+            required={formik.errors.email}
```

Comment:
is there a way to avoid passing formik props to individual attributes? can you evaluate the best approach between `useFormik` and `<Formik>`?

Reply:
Made some changes. Can you please take a look once.